### ISSUE TITLE
Installation failed with Laravel 5.5

### ISSUE THREAD

**Issue by imunew** [amranidev/scaffold-interface#142]
## Installation failed
```bash
$ composer require amranidev/scaffold-interface
Using version ^1.7 for amranidev/scaffold-interface
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install amranidev/scaffold-interface v1.7.15
    - Conclusion: don't install amranidev/scaffold-interface v1.7.1
    - Conclusion: remove laravel/framework v5.5.13
    - Installation request for amranidev/scaffold-interface ^1.7 -> satisfiable by amranidev/scaffold-interface[v1.7.0, v1.7.1, v1.7.15].
    - Conclusion: don't install laravel/framework v5.5.13
    - amranidev/scaffold-interface v1.7.0 requires vinkla/pusher ^2.4 -> satisfiable by vinkla/pusher[2.4.0, 2.5.0].
    - vinkla/pusher 2.4.0 requires illuminate/support 5.1.* || 5.2.* || 5.3.* -> satisfiable by illuminate/support[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.30, v5.1.31, v5.1.41, v5.1.6, v5.1.8, v5.2.0, v5.2.19, v5.2.21, v5.2.24, v5.2.25, v5.2.26, v5.2.27, v5.2.28, v5.2.31, v5.2.32, v5.2.37, v5.2.43, v5.2.45, v5.2.6, v5.2.7, v5.3.0, v5.3.16, v5.3.23, v5.3.4].
    - vinkla/pusher 2.5.0 requires illuminate/support 5.3.* || 5.4.* -> satisfiable by illuminate/support[v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - don't install illuminate/support v5.1.1|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.13|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.16|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.2|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.20|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.22|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.25|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.28|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.30|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.31|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.41|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.6|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.1.8|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.0|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.19|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.21|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.24|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.25|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.26|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.27|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.28|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.31|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.32|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.37|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.43|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.45|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.6|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.2.7|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.3.0|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.3.16|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.3.23|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.3.4|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.0|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.13|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.17|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.19|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.27|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.36|don't install laravel/framework v5.5.13
    - don't install illuminate/support v5.4.9|don't install laravel/framework v5.5.13
    - Installation request for laravel/framework (locked at v5.5.13, required as 5.5.*) -> satisfiable by laravel/framework[v5.5.13].


Installation failed, reverting ./composer.json to its original content.

```

## Environment
- PHP 7.0.14
- laravel/framework 5.5.13

## Cause
`vinkla/laravel-pusher ^2.4` is old. Not supported `illuminate/contracts` and `illuminate/support` 5.5.*.

https://github.com/vinkla/laravel-pusher/blob/2.5.0/composer.json
```json
    "require": {
        "php": "^7.0",
        "illuminate/contracts": "5.3.* || 5.4.*",
        "illuminate/support": "5.3.* || 5.4.*",
        "graham-campbell/manager": "^2.5",
        "pusher/pusher-php-server": "^2.6"
    },
```

https://github.com/vinkla/laravel-pusher/blob/3.0.0/composer.json
```json
   "require": {
        "php": "^7.0",
        "illuminate/contracts": "5.3.* || 5.4.* || 5.5.*",
        "illuminate/support": "5.3.* || 5.4.* || 5.5.*",
        "graham-campbell/manager": "^3.0",
        "pusher/pusher-php-server": "^3.0"
    },
```

**Comment by amranidev** [comment]
@imunew Thank you for opening this issue.

**Comment by amranidev** [comment]
@imunew.

Do you find Pusher/websocket is an interesting feature to be even existed in this project?
what I see is that  vinkla/laravel-pusher is such a pain.

**Comment by amranidev** [comment]
@imunew 

Just Released scaffold-interface v5.5, now it supports laravel 5.5 and requires php7.

Feel free to submit any issue or pull request.

**Comment by imunew** [comment]
@amranidev 
Thank you for the release v1.7.21.
I will install and use this version.
